public/scripts/populateTable.js

```diff
@@ -23,18 +23,38 @@ $(() => {
     }
   }
 
-  const appendDateToTable = (month, day) => {
-    const $heading = $('<th>').attr('scope', 'col').text(monthToString(month) + day);
+  const appendDateToTable = (month, day, hours, minutes) => {
+    let minuteString = minutes > 9 ? minutes : '0' + minutes;
+    let timeString = hours > 12 ? (hours - 12) + ':' + minuteString + ' PM' : hours + ':' + minuteString + ' AM';
+    const $heading = $('<th>').attr('scope', 'col').text(monthToString(month) + day + ' ' + timeString);
     $table.children("thead").children("tr").append($heading);
   }
 
+  const getDropdown = (voteValue) => {
+    let selection = voteValue ? 'Yes' : 'No';
+    if (voteValue === null) {
+      selection = '-';
+    }
+    return $(`<div class="dropdown">
+    <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+      ${selection}
+    </button>
+    <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+      <a class="dropdown-item" href="#">Yes</a>
+      <a class="dropdown-item" href="#">No</a>
+      <a class="dropdown-item" href="#">-</a>
+    </div>
+  </div>`);
+  }
+
```

Comment:
This needs to be completed server side in the .ejs not in the client js - we need to discuss this